### ISSUE TITLE
[SCSS|Sass] Numbers following interpolations in class names bug

### ISSUE THREAD

**Issue by bgriffith** [tonyganch/gonzales-pe#207]
Numbers following interpolation in a class selector currently cause an issue.

This is down to the fact that when interpolation is used, numbers that follow are not considered part of an ident as happens when the class name is non-interpolated.
# 
# Example issue

`.#{$foo}123` will cause an issue while `.bar123` won't.

In the latter example the tree is:

```
 -> stylesheet
 | -> ruleset
 | | -> selector
 | | | -> class
 | | | | -> ident
 | | | |    "bar123"
```

@tonyganch In your opinion what type should the numbers be that follow `#{$foo}` in the first example?

Would it simply be number (Easy to fix) or would it be considered an ident? (More complicated)

```
 -> stylesheet
 | -> ruleset
 | | -> selector
 | | | -> class
 | | | | -> interpolation
 | | | | | -> variable
 | | | | | | -> ident
 | | | | | |    "foo"
 | | | | -> number
 | | | |    "123"
```

**Comment by tonyganch** [comment]
I think it's okay to make it just a number like in your example above.

**Comment by bgriffith** [comment]
@tonyganch Great! I've pushed a [PR fixing this](https://github.com/tonyganch/gonzales-pe/pull/208). Feedback would be great.

**Comment by tonyganch** [comment]
Fixed by merging #208 into `dev`.
